README.md

```diff
@@ -91,6 +91,12 @@ Since Spring Boot 2.1 bean overriding is disabled. If you want to enable it you
 
 JUnit 5 is now enabled by default in the project. Please refrain from using JUnit4 and use the next generation
 
+## Integration Testing
+
+To run integration tests (After PUB-1000 is merged), [Docker](https://www.docker.com) is now required to run integration tests.
+This is due to the use of [TestContainers](https://www.testcontainers.org) to stand up a local Postgres instance for testing.
+Previously we used H2 for this purpose, which did not require docker.
```

Comment:
youll need to incude what setup is needed to run the tests, having docker doesnt mean much in the context of how to run the rests, what images do we need running etc

Reply:
so testcontainers will automatically download and stand up the relevant containers as required, but docker does need to be running on your machine just so it can deal with things properly.